src/lookup/query.ts

```diff
@@ -61,6 +72,22 @@ export type VerifiablePresentationRequest = {
   domain?: string;
 };
 
+interface ParsedVerifiablePresentation extends VerifiablePresentation {
+  verifiableCredential?: VerifiableCredential[];
+}
+
+/**
+ * @deprecated Use RDFJS API
+ */
+export async function query(
```

Comment:
The orders of declaration in the file will be used for the order of appearance in the API docs, so you'll want to move the deprecated functions after the non-deprecated ones.